modules/op-running-a-pipeline.adoc

```diff
@@ -35,8 +35,8 @@ The <pipelinerun_id> in the above command is the ID for the `PipelineRun` that w
 $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-3/01_pipeline/03_persistent_volume_claim.yaml \
     -p deployment-name=vote-ui \
```

Comment:
we don't use `release-tech-preview-3` instead we use `pipelines-1.3`

Please do changes wherever its required
Thank you